rero/rero-ils-ui
# ci-po: improve settings display in detail view

* Displays settings within a matrix for a more effective visual aspect.
* Removes useless component.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

Part of https://github.com/rero/rero-ils-ui/issues/43
To improve settings display in ci-po detail view as asked during the demo.

## How to test?

See detailed views of ci-po.
Check responsiveness, for example with 'no checkout' and 'extended' ci-po.

## Code review check list
